handbook/community.md

```diff
@@ -187,24 +188,110 @@ graph TD;
+- Offer code review and coaching to help get the PR ready to go (see note below)
+- Keep an eye out for any updates or responses.
+
+> Sometimes (typically for Fleet customers), a Fleet team member may add tests and make any necessary changes to merge the PR. 
+
+If everything is good to go, approve the review.
+
+For PRs that will not be merged:
+
+- Thanks the contributor for their effort and explain why the changes won't be merged.
+- Close the PR.
+
+## Updating Docs and FAQ
+
+When someone asks a question in a public channel, it's pretty safe to assume that they aren't the only person that's ever had the same question. In order to make sure the documentation is as useful as possible, the Community team gathers questions and uses them to make a weekly documentation update. 
 
-When community members contribute changes, Fleet’s top priority is to help them feel engaged. We acknowledge contributions as quickly as possible, within 1 business day. We also drive to a resolution (close/merge) as soon as possible, though this may take longer than 1 business day.
+As a general goal, every question should be answered with a link to the Docs and/or result in an update to the Docs. 
 
-### Process
+### Tracking
 
-1. Decide if the change is acceptable (see below). If this will take time, acknowledge the contribution and let the user know that the team will respond. If you choose not to approve a change, first thank the contributor for their interest. Next, encourage them to open an issue or share their ideas before writing more code. Discussions can occur in the #fleet channel of osquery's Slack.
-2. Help the contributor make the content suitable for merging. Ensure appropriate manual and automated testing occurred. Also, confirm that changes to files and documentation are updated. Usually, this is best done with code review and coaching the user. Sometimes Fleet may complete a PR by improving testing and code review.
-3. After reviewing and addressing all the changes a PR needs, a Fleet team member may merge. Before merging, please use your best judgment. Double-check that CI is passing, documentation is updated, and a changes file is created.
-4. Once a PR is approved and merged, thank and congratulate the contributor. Next, share it with the team in the #help-promote channel of Fleet's Slack so they can promote it on social media. Recognized contributors often become project champions.
+When responding to a question or issue in the #fleet channel of the osquery Slack workspace, push the thread to Zapier using the `TODO: Update docs` Zap. This will add information about the thread to the [Slack Questions Spreadsheet](https://docs.google.com/spreadsheets/d/15AgmjlnV4oRW5m94N5q7DjeBBix8MANV9XLWRktMDGE/edit#gid=336721544). In the `Notes` field, you can include any information that you think will be helpful when making weekly doc updates. That may be something like:
```

Comment:
```suggestion
When responding to a question or issue in the [#fleet](https://osquery.slack.com/join/shared_invite/zt-h29zm0gk-s2DBtGUTW4CFel0f0IjTEw#/) channel of the osquery Slack workspace, push the thread to Zapier using the `TODO: Update docs` Zap. This will add information about the thread to the [Slack Questions Spreadsheet](https://docs.google.com/spreadsheets/d/15AgmjlnV4oRW5m94N5q7DjeBBix8MANV9XLWRktMDGE/edit#gid=336721544). In the `Notes` field, you can include any information that you think will be helpful when making weekly doc updates. That may be something like:
```